packages/@azure/servicebus/data-plane/test/streamingReceiverSessions.spec.ts

```diff
@@ -69,10 +69,10 @@ async function beforeEachTest(senderType: ClientType, receiverType: ClientType):
 
   ns = Namespace.createFromConnectionString(process.env.SERVICEBUS_CONNECTION_STRING);
 
-  senderClient = getSenderClient(ns, senderType);
+  senderClient = await getSenderClient(ns, senderType);
   sender = senderClient.getSender();
```

Comment:
Does this also need an await?

Reply:
No, only the senderClient (which can be queueClient or topicClient) needs the await because the corresponding queue/topic needs to be deleted and then created.

The `getSender()` is a call on the client which just gives the sender object for the user to then use to send messages